maths/signum.py

```diff
@@ -3,20 +3,37 @@
 """
 
 
-def signum(num: float) -> int:
+def signum(num) -> int:
     """
     Applies signum function on the number
 
-    >>> signum(-10)
+    Custom test cases:
+    >>> signum(-20)
     -1
-    >>> signum(10)
+    >>> signum(20)
     1
     >>> signum(0)
     0
+    >>> signum("a")
+    Traceback (most recent call last):
+        ...
+    TypeError: '<' not supported between instances of 'str' and 'int'
+    >>> signum([])
+    0
```

Comment:
This should also be a TypeError... https://github.com/TheAlgorithms/Python/pull/10724#issuecomment-1772362052